addons/fastroping/CfgSounds.hpp

```diff
@@ -0,0 +1,12 @@
+class CfgSounds {
+    class ACE_Fastroping_Rope {
+        name = "ACE_Fastroping_Rope";
+        sound[] = {PATHTOF(data\sounds\fastroping_rope.ogg), db0, 1.0};
```

Comment:
Missing quotes (for compatibility with other non-standard tools).
`QUOTE(PATHTOF(path))`